### PR TITLE
Added genre buttons in navigation. Also added employee login button a…

### PR DESCRIPTION
**Pull request by ejporter66**
added employee login at the bottom right and genre buttons in navigation section in the homepage. working on admin dashboard.

### CODE REVIEW

File: src/components/AdminDashboard/Dashboard.js [hunk 1]
```diff
@@ -1,9 +1,29 @@
...
+        {/* Include the necessary components and functionality for updating inventory */}
+      </div>
+      <div>
+        <h2>Monitor Performance</h2>
+        {/* Include the necessary components and functionality for monitoring performance */}
+      </div>
+      <div>
+        <h2>Handle Security Configurations</h2>
+        {/* Include the necessary components and functionality for handling security configurations */}
+      </div>
```
Comment (mlenda000) [inline src/components/AdminDashboard/Dashboard.js:23]: these changes should be made in the Grid component

File: src/components/HomePage/HomePage.js
Comment (mlenda000) [inline src/components/HomePage/HomePage.js:1]: this is handled by the normal login no need for the redundancy but nice work picking up on how to set the useState variables.

File: src/components/HomePage/styles/homepage.scss
Comment (mlenda000) [inline src/components/HomePage/styles/homepage.scss:1]: this goes with the previous content no need for a secondary version of this

File: src/components/Navigation/Navigation.js
Comment (mlenda000) [inline src/components/Navigation/Navigation.js:1]: looks good feel free to style these buttons to better match the site also don't be afraid to add some styling to align them better in the div

File: src/config/fireBaseConfig.js
Comment (mlenda000) [inline src/config/fireBaseConfig.js:1]: please don't upload this if you haven't gotten a warning from github please remove it following these steps:

You can purge a file from your repository's history using either the git filter-repo tool or the BFG Repo-Cleaner open source tool. Note: If sensitive data is located in a file that's identified as a binary file, you'll need to remove the file from the history, as you can't modify it to remove or replace the data.

**Review by mlenda000 — CHANGES_REQUESTED** [review]
Nice work taking the initiative we can definitely use some of these changes just make sure your placing them in the correct files update recommit and we can make sure everything works and is good and then merge it.

**Review by iccugs — CHANGES_REQUESTED** [review]
we can probably get rid of this pull request until all the issues have been addressed insofar as all the extra files and the firebaseconfig file being removed.

**Pull request closed without merging**